src/jit/regset.cpp

```diff
@@ -3320,6 +3320,14 @@ void Compiler::tmpPreAllocateTemps(var_types type, unsigned count)
         tmpCount++;
         tmpSize += size;
 
+#ifdef _TARGET_ARM_
+        if (type == TYP_DOUBLE)
+        {
+            // Adjust tmpSize in case it needs alignment
+            tmpSize += TARGET_POINTER_SIZE;
```

Comment:
> It seems to need reserving the TARGET_POINTER_SIZE of additional temp size for aliging beforehand(The final decision of the stack offset is made in Compiler::lvaAllocateTemps. It is only a temporary space for estimating the size. It may not be necessary.).

I see, I got your point. So it's just a estimation for max size and I also agree that we don't know exact offset at the register allocation phase yet.

LGTM